eth/executionclient/execution_client_test.go

```diff
@@ -149,7 +58,7 @@ Example contract to test event emission:
 const callableBin = "6080604052348015600f57600080fd5b5060998061001e6000396000f3fe6080604052348015600f57600080fd5b506004361060285760003560e01c806334e2292114602d575b600080fd5b60336035565b005b7f81fab7a4a0aa961db47eefc81f143a5220e8c8495260dd65b1356f1d19d3c7b860405160405180910390a156fea2646970667358221220029436d24f3ac598ceca41d4d712e13ced6d70727f4cdc580667de66d2f51d8b64736f6c63430008010033"
 
 func TestFetchHistoricalLogs(t *testing.T) {
-	const testTimeout = 1000 * time.Millisecond
+	const testTimeout = 10000 * time.Millisecond
```

Comment:
You probably need it for debugging, but I'm leaving a comment to not forget: We need to make this lower before merging because 10 sec indicate that something is wrong